pkg/flare/archive_test.go

```diff
@@ -166,6 +167,7 @@ func TestGetWorkloadList(t *testing.T) {
 	assert.Contains(t, string(content), "Name: init-volume ID: e19e1ba787")
 	assert.Contains(t, string(content), "container_id_2")
 	assert.Contains(t, string(content), "Name: init-config ID: 4e0ffee5d6")
+	assert.NotContains(t, string(content), "Name: init-passwd ID: f17539af8d pwd: admin123")
```

Comment:
What do you think about changing from `NotContains(...)` to `Contains(... "pwd: *********")` to show the actual output of the scrub?

Reply:
right, updated.